### ISSUE TITLE
Field validation error messages do not link to specified field.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12415]
## Description
There exists a [Core bug](https://www.drupal.org/project/drupal/issues/3229493) where violations in validating a text field do not create an error message that links correctly to the field in question.  I created a patch but it was never merged or followed up, and recently it was altered and re-rolled as part of the 9.4.x -> 9.5.x upgrade.  

This causes a 508 issue.

It's possible that the underlying bug was fixed by the 9.5 upgrade, and my bug is a dupe, and now the patch interferes and re-breaks error messages.

So the fix for this might be as simple as removing the patch.  On the other hand, it might not -- and as I recall, this took a decent amount of spelunking to find in the first place.

## Acceptance Criteria
- [ ] Validation error messages for text fields link to the involved field.

**Comment by ndouglas** [comment]
Related:
- [Improve the documentation of WidgetBase::errorElement() for mapping violation property paths to form elements](https://www.drupal.org/project/drupal/issues/2027059)
- [No way to set error on a field from an entity constraint validator](https://www.drupal.org/project/drupal/issues/2714347)
- [TextareaWidget::errorElement, field 'arrayPropertyPath' not found.](https://www.drupal.org/project/drupal/issues/2600790)
